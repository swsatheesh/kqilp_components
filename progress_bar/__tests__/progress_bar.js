import React from 'react';
import renderer from 'react-test-renderer';
import ProgressBar from '../progress_bar';

jest.unmock('../progress_bar');

describe('components', () => {
  describe('<ProgressBar />', () => {
    it('should render correctly', () => {
      const Progress = renderer.create(
        <ProgressBar now={10} />
      ).toJSON();
      expect(Progress).toMatchSnapshot();
    });
    it('should render correctly with active stripes', () => {
      const Progress = renderer.create(
        <ProgressBar now={10} striped active labelVisibility>Progress Level</ProgressBar>
      ).toJSON();
      expect(Progress).toMatchSnapshot();
    });
  });
});
