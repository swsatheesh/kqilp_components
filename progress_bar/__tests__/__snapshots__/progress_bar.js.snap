// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <ProgressBar /> should render correctly 1`] = `
<div
  className="progress "
>
  <div
    className="progress-bar  "
    max={100}
    min={0}
    now={10}
    style={
      Object {
        "width": "10%",
      }
    }
  >
    <span
      className=""
    >
      10%
    </span>
  </div>
</div>
`;

exports[`components <ProgressBar /> should render correctly with active stripes 1`] = `
<div
  className="progress "
>
  <div
    className="progress-bar progress-bar-striped active"
    max={100}
    min={0}
    now={10}
    style={
      Object {
        "width": "10%",
      }
    }
  >
    <span
      className=""
    >
      Progress Level
    </span>
  </div>
</div>
`;
